maintainer_reports/README.md

```diff
@@ -0,0 +1,27 @@
+A CLI for querying GraphQL data from GitHub and persisiting it to BigQuery for
```

Comment:
Are we gradually allowing .md instead of .rst ?

Reply:
No, the standard is rst for anything that is part of the Open edX platform.  In this case capturing basic details quickly was prioritized.